xbmc/platform/win32/Win32InterfaceForCLog.cpp

```diff
@@ -20,21 +18,21 @@
 *
 */
 
-#include "utils/log.h"
-
 #ifndef TARGET_WINDOWS
```

Comment:
should be `#if !defined(TARGET_WINDOWS) || !defined(TARGET_WIN10)` for now, might be needed to make a separate logger for uwp(the win10 target) later.